### DISCUSSION TITLE
[错误报告]: 站点朱雀会显示连通失败:cookie失效

### DISCUSSION THREAD

**Discussion by UFO404** [jxxghp/MoviePilot#374] [问题咨询]
### 确认

- [X] 我的版本是最新版本，我的版本号与 [version](https://github.com/jxxghp/MoviePilot/releases/latest) 相同。
- [X] 我已经 [issue](https://github.com/jxxghp/MoviePilot/issues) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经 [Telegram频道](https://t.me/moviepilot_channel) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经修改标题，将标题中的 描述 替换为我遇到的问题。

### 当前程序版本

v1.0.9

### 问题类型

主程序运行问题

### 问题描述

站点朱雀会显示  站点连通失败:cookie失效
实际可以浏览到种子页面也能参与搜索搜到资源
站点连通性测试失败后会导致cookie同步时不同步,只能手动添加站点

**Comment by mjsoftking** [comment]
一样的问题，但是有趣的是我在浏览器覆盖cookie也是不能登录


而且IYUU辅种添加站点也没有需要cookie，不知道和站点本身是不是有什么关系

**Comment by jxxghp** [comment]
朱雀架构特殊，要开仿真
